templates/user/contactForm.tpl

```diff
@@ -23,7 +23,14 @@
 	{include file="controllers/notification/inPlaceNotification.tpl" notificationId="contactFormNotification"}
 
 	{fbvFormSection}
-		{fbvElement type="email" label="user.email" id="email" value=$email size=$fbvStyles.size.MEDIUM required=true}
+		{if $changeEmailPending}
+			<p>
+				{fbvElement type="hidden" id="pendingEmail" value=$changeEmailPending}
+				{translate key="user.pendingEmailChange" pendingEmail=$changeEmailPending}
+				<button type="submit" name="action" value="cancelPendingEmail">Cancel</button>
```

Comment:
(hard-coded "Cancel" -- should be localized)

Reply:
used `{translate key="common.cancel"}`